code/game/objects/structures/deployable_turret.dm

```diff
@@ -16,20 +17,41 @@
 	var/projectile_type = /obj/projectile/bullet/manned_turret
 	var/rate_of_fire = 1
 	var/number_of_shots = 40
-	var/cooldown_duration = 90
+	var/cooldown_duration = 9 SECONDS
 	var/atom/target
 	var/turf/target_turf
 	var/warned = FALSE
 	var/list/calculated_projectile_vars
+	var/overheatsound = 'sound/weapons/sear.ogg'
+	var/firesound = 'sound/weapons/gun/smg/shot.ogg'
+	var/can_be_undeployed = FALSE
+	var/obj/spawned_on_undeploy
+	var/undeploy_time = 3 SECONDS
```

Comment:
Document all this since it is used in subtypes.

Same goes for anything that's not new that you use in subtypes.